packages/core/test/editable-text/editableTextTests.tsx

```diff
@@ -58,6 +58,22 @@ describe("<EditableText>", () => {
             assert.deepEqual(changeSpy.args, [["hello"], [" "], ["world"]]);
         });
 
+        it("calls onChange when escape key pressed and value is reverted", () => {
```

Comment:
your EDIT is exactly what i'm testing though: that onChange is called when escape key is pressed _and_ the value should be reverted to the last confirmed value.